stock_picking_completion/hooks.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def column_exists(cr, args):
+    column_exists_query = """
+        SELECT column_name
+        FROM information_schema.columns
+        WHERE table_name=%(table)s AND
+        column_name=%(column)s;
+    """
+    cr.execute(column_exists_query, args)
+    return cr.fetchone()
+
+
+def setup_move_line_completion(cr):
+    table = "stock_move_line"
+    column = "completion"
+    args = {"table": table, "column": column}
+    if column_exists(cr, args):
```

Comment:
Nitpick, `args` is not obvious
```suggestion
    if column_exists(cr, table, column):
```